docs/project_open_energy_dashboard/about.md

```diff
@@ -0,0 +1,11 @@
+---
+id: about
+title: Open Energy Dashboard
+custom_edit_url: null
+---
+![Alt](OED.png) 
+
+## Overview
+
+The Open Energy Dashboard (OED) is a crucial open-source project that facilitates the visualization of energy information through a web-based application. Its significance lies in promoting collaboration and knowledge-sharing to advance common sustainability goals. Unlike other proprietary dashboards, OED is free, portable, and driven by user needs, embodying principles such as perpetual freedom, shared advancements, and user-centric design. The typical user, ranging from energy managers to sustainability professionals, benefits from its user-friendly interface and diverse functionalities, allowing easy management of energy data without the need for external support.
+
```

Comment:
Please add as much detail in the ### Information section as you can. It doesn't need to have everything, but there are things you know (repo URL, client, tech lead, adoption date, technologies used, type, license).